digitalstrategie/assets/scss/_variables.scss

```diff
@@ -242,3 +242,22 @@ $accordion-icon-transform:                rotate(-45deg);
 
 $accordion-button-icon:     url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='#{$text-gray}'><path d='M8 0a1 1 0 0 1 1 1v6h6a1 1 0 1 1 0 2H9v6a1 1 0 1 1-2 0V9H1a1 1 0 0 1 0-2h6V1a1 1 0 0 1 1-1z'/></svg>");
 $accordion-button-active-icon:  url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='#{$text-gray}'><path d='M8 0a1 1 0 0 1 1 1v6h6a1 1 0 1 1 0 2H9v6a1 1 0 1 1-2 0V9H1a1 1 0 0 1 0-2h6V1a1 1 0 0 1 1-1z'/></svg>");
+
+@font-face {
+  font-family: 'BerlinType';
+  src: url('../fonts/BerlinTypeWeb-Regular.eot'); /* IE9 Compat Modes */
+  src: url('../fonts/BerlinTypeWeb-Regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/BerlinTypeWeb-Regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/BerlinTypeWeb-Regular.woff') format('woff'), /* Pretty Modern Browsers */
+       url('../fonts/BerlinTypeOffice-Regular.ttf')  format('truetype'), /* Safari, Android, iOS */
+}
+
+@font-face {
```

Comment:
this is a file for BS overwrites, I would add fonts in seperate file like in Opin